src/interfaces/forms.d.ts

```diff
@@ -88,9 +105,13 @@ export interface IInputFieldProps extends IInputProps {
   inline?: boolean;
   description?: React.ReactNode;
   fieldClassName?: string;
+  ref?:
+    | React.Ref<HTMLButtonElement>
```

Comment:
it should be React.Ref<HTMLInputElement> :)

Reply:
Also you can fix it only in `IInputFieldProps`. Ref is not needed here because of `IInputFieldProps extends IInputProps` :D